bin/pycbc_inspiral

```diff
@@ -92,10 +92,13 @@ parser.add_argument("--cluster-function", choices=["findchirp", "symmetric"],
 parser.add_argument("--cluster-window", type=float, default = -1,
                     help="Length of clustering window in seconds."
                     " Set to 0 to disable clustering.")
-parser.add_argument("--maximization-interval", type=float, default=0,
+parser.add_argument("--maximization-interval", nargs='+', type=float, default=0,
```

Comment:
Why this change? (If the answer is not "This is a mistake" the help text would need updating)